packages/amplify-provider-awscloudformation/src/graphql-transformer/utils.ts

```diff
@@ -327,3 +327,15 @@ export function throwIfNotJSONExt(stackFile: string) {
     throw new Error(`Invalid extension ${extension} for stack ${stackFile}`);
   }
 }
+
+const emptyBuildDir = (directory: string) => {
```

Comment:
is the purpose of this function to empty everything except the tsconfig.resource.json file? If so, I'd rename this function to `emptyBuildDirRetainTsconfig` But also, why does the tsconfig need to be retained? Seems like all files in a build directory should be ephemeral